src/Translation_unit.ml

```diff
@@ -17,11 +17,10 @@ open Migrate_ast
 
 (** Operations on translation units. *)
 type 'a t =
-  { input: string -> In_channel.t -> 'a * (string * Location.t) list
```

Comment:
Note that `input` and `parse` are different in the reason support. Is removing `input` necessary for this change?

Reply:
Ah, OK sorry about this.  I did not see it while compiling.  Is the reason support in another branch or another repo?

Removing input is not necessary for this change, I will make another version where this does not happen.

Reply:
I see it now, it's:
```
make reason
```
which is not built by default (and btw, it is currently broken as I missed the no_translation change in f00efadca380e94f38db320d8ac61d52b0c97815).

Shall the target  *reason* be added to the default make target?

Reply:
It could be added to the default target. It isn't now only to avoid waiting for the extra link step on each build when developing. Breaking compilation of the reason stuff has been rare. But mainly it has not been tested for some time, last time with reason < v2, so I'm unsure if it works reasonably well, or if anyone is using it.

Reply:
Understood, thank you.

I updated the commit to not break the compilation this time.